Major spell check and moderate wording updates (#4)
saint11/DeadWeight: source/index.md

# Core mechanics

## Game Structure
A game of **Dead Weight** runs a bit differently than most tabletop RPGs. It follows a strong structure, with two main states.

- [Expedition](#expedition-phase)
- [Interlude](#interlude)

The game usually starts in the **Expedition** phase, with the adventurers raiding a dungeon or some other terrible place in the **Blacksalt Isle**.

## Dice Notation
Like many tabletop RPGs, **Dead Weight** uses dice notation to specify when and how you should roll your dice. Most die rolls are common six-sided dice, unless specified otherwise.

When you see **d4**, **d6**, **d8**, **d10**, **d12** and **d20**, this is referring to a die of that size; d4 is a four-sided die and d20 is a twenty-sided die. Let's review all styles of dice notations we will use in this book:

### Adding dice (2d6) {.unlisted}
When you see **3d6**, that means roll **three six-sided dice** and add the result.

### Usage dice (Ud8) {.unlisted}

> Usage dice track consumable use. For instance, **Throwing Daggers (Ud8)** have a d8 usage die. Every time the consumable is used, the usage dice are rolled: if the result is a 1 or 2, that die is downgraded to the (next?) smallest value in the following chain:

Author: Wasn't sure if the chain should exist for a reason if it's not going to the next smallest, but always the smallest. I think next was intended here, will add if you give the go-ahead.

Reviewer: yes, next was intended!
```suggestion
Usage dice track consumable use. For instance, **Throwing Daggers (Ud8)** have a d8 usage die. Every time the consumable is used, the usage dice are rolled: if the result is a 1 or 2, that die is downgraded to the next smallest value in the following chain:
```

Note that any **fatigue** generated by that dice should be received by the helping character only.

Negative consequences of a failed task are also suffered by the helping character, where applicable, but not the **fatigue** rolled by the main character.

### Extra Effort

If the player really wants to succeed, it is possible to add one more dice to the ***available dice*** pool by causing the character one **fatigue** point before the roll, as long the player announces that **before** rolling the dice. A character cannot spend **fatigue** if it would cause the character to receive a wound as consequence of doing so.

Players may (not?) use extra effort after rolling the dice.

<pagebreak>

If the player is already going for the **extra effort**, rolling a 1 or a 4 will not cause **fatigue**, since they are already receiving that point.

Helping characters can also use the ***extra effort*** rule, rolling two dice instead of one, as long as they announce that before rolling.

### Zero Dice?

> If for any reason you need to roll zero dice, you roll 2 dice instead and keep the lowest number. Rolling less than zero dice is also possible: add dice to ensure you have 2 total dice being rolled. For instance, to roll -2 dice when you have only 1 dice available, roll 4 dice and choose the worst. I'm pretty sure that test will fail, though, sorry.

Author: I want to make sure this is accurate.

Reviewer: I think this part is a bit confusing, I'll try to reword it for more clarity.

```suggestion
If, for any reason, you need to roll zero dice, you should roll 2 die instead and take the lowest number instead. Rolling less than zero dice is also possible: roll 2 die and add one extra dice for each value below zero. For instance, when rolling -3 die, roll 5 die (2+3) and choose the worst. I'm pretty sure that test will fail, though, sorry.
```

[Converting distances]

## Trackers
Also known as a *progress clock* in other systems, a **tracker** is just one or more circles divided into segments that the *GM* uses to track the progression of something. Every time a tracker is advanced, one segment must be filled. When the circle is completely filled, the tracker is triggered and something happens. There are 3 main trackers that are always in play:

- **Decay tracker**: Represents the toll that dungeon delving takes on the **adventurers**. Causes hunger, exhaustion and other bad things to the players. Advances every turn.
- **Doom tracker**: There's one for each dungeon, representing the dungeon "waking up" to the **adventurers**' presence and reacting.

> - **Omens tracker**: There's one for the whole campaign. Represents the **adventurers**' terrible luck and their inevitable doom in **Blacksalt island**. Every time it gets completed, something really bad happens. (???? Are these two reversed?)

Author: I wasn't sure about this - it seems like these two values are reversed, but could be completely wrong.

Reviewer: I'm considering renaming it to "Bad Omens" tracker.
Honestly my logic is that the "Doom" just sounds cooler, and I wanted the cool thing to happen more often. But I might consider renaming both in the future.

```suggestion
- **Bad Omens tracker**: There's one for the whole campaign. Represents the **adventurers**' terrible luck and their inevitable doom in **Blacksalt island**. Every time it gets completed, something really bad happens.
```
